package.json

```diff
@@ -4,13 +4,13 @@
   "scripts": {
     "ng": "ng",
     "start": "npm run build:misc && ng serve",
-    "start:local": "npm run build:misc && ng serve --configuration=local",
+    "start:mocky": "npm run start -- --configuration=mocky",
```

Comment:
why are we using the mocky configuration?

Reply:
purely so that during local development FE dont have to reply on BE updates to an environment and so that FE can quickly stub out a BE endpoint. or perhaps if an environment is down etc is unblocks the FE